src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
I suppose what we could do is, if we see double quotes, treat that as a string and then format that accordingly.

But since this really is intended only for mock overrides and should be generated only by automated tooling, I'd rather hold off on complicating the logic if possible.